_layouts/default.html

```diff
@@ -1,12 +1,23 @@
+{% assign relroot = '' %}
+{% if page.lang != 'nl' %}
+{%   assign relroot = '../' %}
+{% endif %}
 <!DOCTYPE html>
 <!-- generated at {{ site.time }} -->
 <html lang="{{page.lang}}">
 {% include head.html %}
 <body>
-    {% include header.html %}
-    <article>
+  {% include header.html %}
```

Comment:
Zelfde suggestie als in https://github.com/minvws/nl-covid19-notification-app-community-website/pull/21

Namelijk de `<h1>` met de titel van het artikel moet binnen de `<main>` sectie vallen; niet erbuiten.